middleware/withShop.js

```diff
@@ -1,3 +1,5 @@
+const TEST_COOKIE_NAME = 'shopifyTestCookie';
```

Comment:
I wasn't sure how to share constants between the middleware and routes, so I have two separate constants. I'd prefer to share, any ideas on how to do that idiomatically?

Reply:
```
// constants.js
module.exports.TEST_COOKIE_NAME = 'shopifyTestCookie';
```

```
// middleware/withShop.js
const {TEST_COOKIE_NAME} = require('../constants');
```